MAINT: Align masked with normal ufunc loops (#19259)
numpy/numpy: doc/release/upcoming_changes/19259.c_api.rst

Masked inner-loops cannot be customized anymore
-----------------------------------------------
The masked inner-loop selector is now never used.  A warning
will be given in the unlikely event that it was customized.

We do not expect that any code uses this.  If you do use it,

> you must unset unset the selector on newer NumPy version.

Reviewer: ```suggestion
you must unset the selector on newer NumPy version.
```

Please also contact the NumPy developers, we do anticipate
providing a new, more specific, mechanism.

> This change will not affect the results of operations, since
> the fallback (which is always used internally) will handle
> the operation equivalently, the customization was a planned
> feature to allow for faster masked operation.

Reviewer: Shorten
```suggestion
The customization was part of a never-implemented feature to allow
for faster masked operations.
```
